RevenueCat/Editor/RevenueCatPostInstall.cs

```diff
@@ -0,0 +1,39 @@
+using System.IO;
+using UnityEngine;
+using UnityEditor;
+using UnityEditor.Callbacks;
+using UnityEditor.iOS.Xcode;
+ 
+namespace Editor
+{
+    public static class XcodeSwiftVersionPostProcess
+    {
+        [PostProcessBuild(999)]
```

Comment:
Just curious, is this the priority?

Reply:
Yes it is :) It ensures that this script will be executed after all other post-build scripts.

Reply:
I added a comment to document this, in the follow-up PR https://github.com/RevenueCat/purchases-unity/pull/85/files#diff-845909e33737cf286aa2e6eb1f3305bddeb055257486dc093760090a7b658a54R11